test/gtest/helpers.cpp

```diff
@@ -128,7 +128,7 @@ P4CTestEnvironment::P4CTestEnvironment() {
     // XXX(seth): We should find a more robust way to locate these headers.
     _coreP4 = readHeader("p4include/core.p4");
     _v1Model = readHeader("p4include/v1model.p4", true,
-                          "V1MODEL_VERSION", P4V1::V1Model::instance.version);
+                          "V1MODEL_VERSION", 20200408);
```

Comment:
Shouldn't this be using getV1ModelVersion?

Reply:
No, getV1ModelVersion extracts the version from an existing loaded program, but here we haven't loaded any program yet. What we are doing is preparing to load the program using this specific version of v1model.

Reply:
I do not quite understand the versioning. Where does 20200408 come from? Last change to the v1model.p4 file?